src/medImageIO/itkDataImageReaderBase.cpp

```diff
@@ -42,6 +42,9 @@ bool itkDataImageReaderBase::canRead (const QString& path)
     if (this->io.IsNull())
         return false;
 
+    // Avoid to display log of each metadata not read by itk::ImageIOBase
+    this->io->SetGlobalWarningDisplay(false);
```

Comment:
Removes invasive pop-up about metadata in Logs (and pop-up? cf @LoicCadour ) during import of data or pipeline.